doc/managing-wallets.md

```diff
@@ -0,0 +1,94 @@
+# Managing the Wallet
+
+## 1. Backing Up and Restoring The Wallet
+
+### 1.1 Creating the Wallet
+
+Since version 0.21, Bitcoin Core no longer has a default wallet.
+Wallets can be created with the `createwallet` RPC. More information about this command may be found by running `bitcoin-cli help createwallet`.
+
+The following command, for example, creates a descriptor wallet:
+
+`$ bitcoin-cli -named createwallet wallet_name="wallet-01" descriptors=true`
+
+In the GUI, the `Create a new wallet` button is displayed on the main screen when there is no wallet loaded. Alternatively, there is the option `File` ->`Create wallet`.
+
+The `descriptors` parameter can be omitted if the intention is to create a legacy wallet.
+
+For now, the default type is the legacy wallet, but that should change in the near future.
+
+By default, wallets are created in the `wallets` folder of the data directory, which varies by operating system, as shown below. The user can change the default by using the `-datadir` or `-walletdir` initialization parameters.
+
+| Operating System | Default wallet directory                                    |
+| -----------------|:------------------------------------------------------------|
+| Linux            | `/home/<user>/.bitcoin/wallets`                             |
+| Windows          | `C:\Users\<user>\AppData\Roaming\Bitcoin\wallets`           |
+| macOS            | `/Users/<user>/Library/Application Support/Bitcoin/wallets` |
+
+### 1.2 Encrypting the Wallet
```

Comment:
Perhaps a section on unlocking the wallet? There's the `walletpassphrase` RPC, and the GUI will prompt to unlock when needed.

Reply:
Will re-ACK after this!

Reply:
Section added.